CODEOWNERS

```diff
@@ -66,14 +66,66 @@ Dockerfile* @cilium/build
 /daemon/cmd/state.go @cilium/endpoint
 /daemon/cmd/sysctl_linux.go @cilium/bpf
 /Documentation/ @cilium/docs-structure
-/Documentation/cmdref @cilium/nonexistantteam
-/Documentation/bpf.rst @cilium/bpf
-/Documentation/contributing/ @cilium/contributing
-/Documentation/envoy/ @cilium/proxy
-/Documentation/gettingstarted/hubble-enable.rst @cilium/hubble
-/Documentation/gettingstarted/hubble-install.rst @cilium/hubble
-/Documentation/gettingstarted/hubble.rst @cilium/hubble
-/Documentation/hubble.rst @cilium/hubble
+/Documentation/_static/ @cilium/docs-structure
+/Documentation/api.rst @cilium/agent @cilium/docs-structure
+/Documentation/beta.rst @cilium/docs-structure
+/Documentation/bpf.rst @cilium/bpf @cilium/docs-structure
+/Documentation/check-build.sh @cilium/docs-structure
+/Documentation/check-cmdref.sh @cilium/docs-structure
+/Documentation/check-crd-compat-table.sh @cilium/docs-structure
+/Documentation/check-examples.sh @cilium/docs-structure
+/Documentation/cmdref/ @cilium/nonexistantteam
```

Comment:
What will happen to this `nonexistantteam`?

Reply:
It will continue to not exist :smile: 

See https://github.com/cilium/cilium/commit/ba5f19633b34b5fe50d77dec7b3580098720fb6f for context.